rateLimits/messageRateLimits.md

```diff
@@ -43,6 +43,12 @@ All Bandwidth messaging products are rate limited in some fashion.  There are va
 
 ### Sample Message Rate Limit Response
 
+You will receive this error response if:
+
+* Your **burst** rate is exceeded
+* Your **per number** MPS is exceeded (only with queueing disabled)
+
+
```

Comment:
A few grammar and clarity changes here:

"Your **burst** rate is exceeded."
"Your **per number** messages per second (MPS) is exceeded (only with queueing disabled)."

Reply:
done